integration-tests/client/chainlink_toml_builder.go

```diff
@@ -0,0 +1,101 @@
+package client
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+)
+
+// TOMLBuilder helps facilitate cleanly building Chainlink settings TOMLs for different tests
+type TOMLBuilder struct {
+	tomlBuilder strings.Builder
+}
+
+// NewTOMLBuilder creates raw TOMLBuilder
+func NewTOMLBuilder() *TOMLBuilder {
+	return &TOMLBuilder{}
+}
+
+// NewDefaultTOMLBuilder shortcut to create TOML with defaults
+func NewDefaultTOMLBuilder() *TOMLBuilder {
+	return NewTOMLBuilder().AddGeneralDefaults()
+}
+
+// NewDefaultNetworksTOMLBuilder shortcut to create TOML with defaults and network settings
+func NewDefaultNetworksTOMLBuilder(networks ...*blockchain.EVMNetwork) *TOMLBuilder {
+	return NewTOMLBuilder().AddGeneralDefaults().AddNetworks(networks...)
+}
+
+// String builds the string value of the TOML to pass to config
+func (t *TOMLBuilder) String() string {
+	return t.tomlBuilder.String()
+}
+
+var defaultTOML = `RootDir = './clroot'
```

Comment:
nit/ Could put these toml strings in files to [embed](https://pkg.go.dev/embed) 🤷

Reply:
Ooo, I was unaware of that package. Will make a ticket.